From fec4c628f0935a1334a861134b893eb55db9289f Mon Sep 17 00:00:00 2001
From: Dmitry Shifrin <dmitry.shifrin@cogentembedded.com>
Date: Fri, 23 Sep 2016 15:50:01 +0300
Subject: [PATCH 1/4] vsp-renderer: initialize v4l2_mbus_framefmt variable

original commit: github.com/renesas-rcar/weston.git
                 f392dbc274d0f3ce5b30c90ce3d40190ff3b79e1
---
 src/vsp-renderer.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/vsp-renderer.c b/src/vsp-renderer.c
index 98abdc8..3d905ec 100644
--- a/src/vsp-renderer.c
+++ b/src/vsp-renderer.c
@@ -612,7 +612,7 @@ static int
 vsp_set_output(struct vsp_device *vsp, struct vsp_renderer_output *out)
 {
 	int i;
-	struct v4l2_mbus_framefmt format;
+	struct v4l2_mbus_framefmt format = { 0 };
 
 	DBG("Setting output size to %dx%d\n", out->base.width, out->base.height);
 
@@ -791,7 +791,7 @@ vsp_set_alpha(struct media_entity *entity, float alpha)
 static int
 vsp_comp_setup_inputs(struct vsp_device *vsp, struct vsp_input *input, int enable)
 {
-	struct v4l2_mbus_framefmt format;
+	struct v4l2_mbus_framefmt format = { 0 };
 	struct vsp_media_pad *mpad = &input->input_pads;
 	struct vsp_scaler *scaler = input->use_scaler;
 	struct vsp_surface_state *vs = input->input_surface_states;
-- 
1.9.1

